src/content/docs/agents/python-agent/getting-started/poetry-break.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: 'Poetry Break'
+---
+
+but they had not rested five minutes when a glare of white letters started from the dark, as vivid as spectres—the air swarmed with Catherines; and rousing myself to dispel the obtrusive name, I discovered my candle wick reclining on one of the antique volumes, and perfuming the place with an odour of roasted calf-skin.
```

Comment:
@bradleycamacho I believe the two stanzas are reversed here. Can you review and fix? Thanks!

Reply:
Flipped the stanzas, nice catch!